src/main/java/org/primefaces/component/datepicker/DatePickerRenderer.java

```diff
@@ -54,12 +54,38 @@
 
     @Override
     public void encodeEnd(FacesContext context, UIComponent component) throws IOException {
-        DatePicker datePicker = (DatePicker) component;
+        DatePicker datepicker = (DatePicker) component;
```

Comment:
why did you rename? its actually datePicker

Reply:
Its mixed throughout the file with datepicker and datePicker i will fix it every where to be datePicker.